Scripts/Miscellaneous/Fake_news_web/Requirements.txt

```diff
@@ -0,0 +1,4 @@
+language: python 3.7
```

Comment:
add used module list in requirements.txt with help of pip freeze

```
pip freeze list>requirements.txt
```

Reply:
ok will do

Reply:
As per your request module list has been added in requirements.txt using the command
pip freeze list>requirements.txt